### PR TITLE
Added External IPAM headline feature

### PR DESCRIPTION
**Pull request by lzap**

### CODE REVIEW

File: _includes/manuals/nightly/1.2_release_notes.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+* External IPAM support through new Smart Proxy API. Subnets can be now associated with brand new IPAM implementation which offloads IP address management to external entities. An experimental implementaion for phpIPAM is available as Smart Proxy plugin.
```
Comment (tbrisker) [inline _includes/manuals/nightly/1.2_release_notes.md]: Can you please add a subheading (`####`) for this like we had in previous manuals for headline features? also please add a link(s) for more information - perhaps to the discourse discussion and/or the relevant manual section

**Comment by lzap** [comment]
Rebased.

File: _includes/manuals/nightly/1.2_release_notes.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### External IPAM
+
+New Smart Proxy External IPAM API allows implementation of IPAM providers. Subnets can be now associated with IPAM plugins providing IP address management capabilities via external entities. An experimental implementaion for phpIPAM is available as a Smart Proxy plugin.
```
Comment (tbrisker) [inline _includes/manuals/nightly/1.2_release_notes.md]: Please add a link for users interested in more information, eg to the discourse discussion about it

**Comment by lzap** [comment]
Will do tomorrow morning got a long meeting and now I have an errand.

**Comment by lzap** [comment]
Rebased, added link to the smart proxy plugin.

**Review by tbrisker — APPROVED** [review]
Thanks @lzap !

**Pull request merged**